tests/functional/010_nfscV3.js

```diff
@@ -232,9 +232,19 @@ async.waterfall([
                             assert.strictEqual(err, null);
                             assert.strictEqual(typeof(object), 'object');
                             assert.strictEqual(obj_attrs.type, mnt.NF3LNK);
-                            done(next, null);
+                            done(next, null, object);
                         });
         }),
+    (symlink, next) =>
+        describeIt('should read the symbolic link', done => {
+            mnt.readlink(symlink, (err, path, obj_attrs) => {
+                assert.strictEqual(err, null);
+                assert.strictEqual(typeof(obj_attrs), 'object');
+                assert.strictEqual(obj_attrs.mode, 0o777);
+                assert.strictEqual(path, 'foo-bar-baz');
+                done(next, null);
+            });
+        }),
```

Comment:
just testing the nominal case here. More complete tests of this module will come with the server as features show up.